docs/modules/reference/pages/ticketing/ticketer/jira.adoc

```diff
@@ -259,12 +272,12 @@ custom checkbox                customfield_10100    true       array <6>
 custom datepicker              customfield_10101    true       date
 ----
 
-<1> Defined components are `core`, `service`, `web`.
-<2> Defined versions are `1.0.0` and `1.0.1`.
-<3> Defined issue types are `Bug` and `Task`.
-<4> Defined priorities are `Major` and `Minor`.
-<5> Defined projects are `NMS` and `HZN`.
-<6> Defined options are `yes`, `no`, and `sometimes`.
+. Defined components are `core`, `service`, and `web`.
+. Defined versions are `1.0.0` and `1.0.1`.
+. Defined issue types are `Bug` and `Task`.
+. Defined priorities are `Major` and `Minor`.
+. Defined projects are `NMS` and `HZN`.
+. Defined options are `yes`, `no`, and `sometimes`.
 
```

Comment:
Not sure this section is needed as the following example re-states these.

Reply:
In that case, should we completely remove this example?

Reply:
Should we keep the output example, but not the legend that explains everything?

Reply:
```suggestion

```